public/app/features/plugins/admin/components/Grid.tsx

```diff
@@ -13,8 +13,9 @@ export const Grid = ({ children }: Props) => {
     <div
       className={css`
         display: grid;
```

Comment:
How is the support for `grid` layout in all browsers? Can we use it e.g. in IE?

Reply:
I don't think we support IE since v8 as we now bundle as ES6. https://github.com/grafana/grafana/pull/32573

For the browsers we target grid support is great. https://caniuse.com/?search=grid